bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestListTables.java

```diff
@@ -85,8 +88,34 @@ public void testTableNames() throws IOException {
         Assert.assertTrue(ArrayUtils.contains(tableList, tableName2));
       }
 
-      checkColumnFamilies(admin, tableName2);
+      {
+        TableName[] tableList = admin.listTableNames(Pattern.compile("list_table1-.*"));
+        Assert.assertTrue(ArrayUtils.contains(tableList, tableName1));
+        Assert.assertFalse(ArrayUtils.contains(tableList, tableName2));
```

Comment:
Failures in our integration tests often leave behind unexpected tables.  I don't want one test to fail for a valid reason and then subsequent tests to fail because the tables were left over.